Makefile

```diff
@@ -60,7 +60,7 @@ quick-stack: ## unoptimized build and tests with stack
 	  --system-ghc --flag 'pandoc:embed_data_files' \
 	  --fast \
 	  --test \
-	  --test-arguments='-j --hide-successes --ansi-tricks=false $(TESTARGS)'
+	  --test-arguments='-j32 --hide-successes --ansi-tricks=false $(TESTARGS)'
```

Comment:
I keep forgetting to remove this, but apparently my version of cabal/stack/ghc/etc. doesn't accept a `-j` here, it insists on having a number.

Reply:
I changed this to `-j4`, it is what other similar flags in this Makefile use, and moved it into a separate commit